### DISCUSSION TITLE
[ramda]@0.29.6 produces errors

### DISCUSSION THREAD

**Discussion by char0n** [DefinitelyTyped/DefinitelyTyped#67032] [Issues with a @types package]
More information in:
- https://github.com/ramda/ramda/issues/3415#issuecomment-1757260119
- https://github.com/swagger-api/swagger-js/issues/3176

Seems like every project that uses @types/ramda@0.29.6 either directly or indirectly (as transitive dep) is affected.

[a comment by an automated account is not shown]

**Comment by Harris-Miller** [comment]
This issue is not tied to the `0.29.1` of ramda, but the `0.29.5` release of `types-ramda` (see https://github.com/ramda/types/releases/tag/v0.29.5)

`@types/ramda` is marked as requiring at least typescript@5 (see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L31)). The error shown in https://github.com/swagger-api/swagger-js/issues/3176 is because of being on typescript@<5. Update typescript will fix
